nad

```diff
@@ -920,7 +928,14 @@ function setup_reverse() {
       } else {
         console.log("configuration error for reverse: no check found (hostname mismatch?)");
       }
-  });
+  }
+
+  circapi.setup(auth_token, 'nad', api_options);
+  if (check_bundle_id == null) {
+      circapi.get('/check_bundle?f_type=json:nad&f_target=' + hostname, null, circapi_cb);
+  else {
+      circapi.get('/check_bundle/' + check_bundle_id, null, circapi_cb);
```

Comment:
Fixed.. my bad.


--
riley

On Thu, Jun 2, 2016 at 1:56 PM, matt maier <notifications@github.com> wrote:

> In nad <https://github.com/circonus-labs/nad/pull/46#discussion_r65587613>
> :
>
> > @@ -920,7 +928,14 @@ function setup_reverse() {
> >        } else {
> >          console.log("configuration error for reverse: no check found (hostname mismatch?)");
> >        }
> > -  });
> > +  }
> > +
> > +  circapi.setup(auth_token, 'nad', api_options);
> > +  if (check_bundle_id == null) {
> > +      circapi.get('/check_bundle?f_type=json:nad&f_target=' + hostname, null, circapi_cb);
> > +  else {
> > +      circapi.get('/check_bundle/' + check_bundle_id, null, circapi_cb);
>
> missing '}' before the else on 936, throws syntax error.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/circonus-labs/nad/pull/46/files/044deee95c2650e1f586158327bc9a956213877a#r65587613>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/ACxODVA_yRf5Rd22z-NYQR1QJGR0FQsJks5qHxlWgaJpZM4Ir8qO>
> .
>